src/BasisTheory.net/Tokens/TokenClient.cs

```diff
@@ -14,6 +14,10 @@ namespace BasisTheory.net.Tokens
 {
     public interface ITokenClient
     {
+        dynamic Tokenize(dynamic tokens, RequestOptions requestOptions = null);
```

Comment:
Using dynamic for the parameter and response may cause some challenges with integration. I believe the `Post` and `PostAsync` calls include JsonDeserializing the response, so the result will likely be one of `JObject` or `JArray`. Should we consider overloads that support either of a list or object of data and then return JObject or JArray as appropriate. Alternatively, we could exclude JSON deserialization of the response on these requests so that the user can interact with an unaltered response body.

Reply:
@koalaty-code I updated the response to be a JToken which should reflect the data type returned from this method.